setup.py

```diff
@@ -0,0 +1,4 @@
+import setuptools
+
+if __name__ == "__main__":
+    setuptools.setup()
```

Comment:
This would be needed if you want to have a Python library with common functions into a package. In that case, you also need a `setup.cfg` like in the [pvae repo](https://github.com/pivlab/pvae/blob/main/setup.cfg), and some other setup.
Since you're not using these now, I would remove it from this PR, and add it when needed.

Reply:
ok!